scripts/exglobal_prep_land_obs.py

```diff
@@ -17,7 +17,10 @@
 
     # Take configuration from environment and cast it as python dictionary
     config = cast_strdict_as_dtypedict(os.environ)
+    HH = os.environ.get("CDATE")[8:]
 
     # Instantiate the land prepare task
     LandAnl = LandAnalysis(config)
-    LandAnl.prepare_IMS()
+    LandAnl.prepare_GTS()
+    if HH == '18':
```

Comment:
@CoryMartin-NOAA I think you are thinking of this. 
```suggestion
    if LandAnl.runtime_config.cyc == '18':
```

Reply:
Made the changes. Thanks @aerorahul